### PR TITLE
Feat: Wishlist page

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added add to wishlist and Remove from wishlist functionality
This works on product page,  cart listing & wishlist  page

Not to check:
filters
quantity on cart listing page

### CODE REVIEW

File: src/Components/Wishlist/WishlistPage.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import "./Wishlist.css"
+
+const WishlistPage = () => {
+
+    // const { state: {cart} } = useCart();    
+    const { stateWishlist: {wishlist}, dispatchWishlist } = useWishlist();
+    
+    const { dispatch } = useCart();
+
+    console.log("wishlist", wishlist);
```
Comment (Shubhamkr2610) [inline src/Components/Wishlist/WishlistPage.jsx]: always try to remove the console.log  part before pushing the code

File: src/Components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
 import React, { useState } from 'react'
 import { Link } from 'react-router-dom'
 import { useCart } from '../../context/Cart-context'
+import { useWishlist } from '../../context/Wishlist-context'
 import { ShoppingCartIcon, UserProfile, WishlistIcon } from '../../images/Svg'
 import './Navbar.css'
```
Comment (Shubhamkr2610) [inline src/Components/Navbar/Navbar.jsx:6]: Appreciation: good way of importing files

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to merge

**Review by archit7448 — APPROVED** [review]
UI looks good , code structure is proper looks good to merge

**Pull request merged**